lib/board.rb

```diff
@@ -0,0 +1,58 @@
+require 'pry'
+class Board
+  attr_accessor :cells
+
+  def reset!
+    @cells = Array.new(9," ")
+  end
+
+  def initialize
+    @cells = Array.new(9, " ")
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.